### ISSUE TITLE
Fatal error: Call to a member function getTitle() on null

### ISSUE THREAD

**Issue by nspyke** [lekoala/silverstripe-debugbar#83]
I'm getting this error.
I'm using PHP 5.6
Mac OSX
Apache 2.4
Silverstripe 3.6.1

I ran multiple flushes, with no change.
Uninstall and flush again, and the site works again.

```
Fatal error: Call to a member function getTitle() on null in .../debugbar/code/DebugBarSilverStripeCollector.php on line 174
--

1 | 0.0006 | 240496 | {main}( ) | ../main.php:0
2 | 4.7042 | 7525160 | Director::direct( ) | ../main.php:191
3 | 131.4706 | 23252328 | RequestProcessor->postRequest( ) | ../Director.php:183
4 | 131.4707 | 23252552 | DebugBarRequestFilter->postRequest( ) | ../RequestProcessor.php:42
5 | 131.4707 | 23252672 | DebugBar::renderDebugBar( ) | ../DebugBarRequestFilter.php:57
6 | 131.4707 | 23254512 | DebugBar\JavascriptRenderer->render( ) | ../DebugBar.php:226
7 | 131.4707 | 23254808 | DebugBar\JavascriptRenderer->getJsInitializationCode( ) | ../JavascriptRenderer.php:981
8 | 131.4708 | 23255144 | DebugBar\JavascriptRenderer->getJsControlsDefinitionCode( ) | ../JavascriptRenderer.php:1015
9 | 131.4710 | 23268080 | DebugBarSilverStripeCollector->getWidgets( ) | ../JavascriptRenderer.php:1059
```

Any ideas on how to fix?

**Comment by robbieaverill** [comment]
Hi @nspyke, what are you trying to do when this message is shown?

**Comment by nspyke** [comment]
Load the home page of my site

**Comment by robbieaverill** [comment]
Thanks. Can you please provide the contents of your composer.json file?

**Comment by nspyke** [comment]
```
{

  "require": {
    "php": ">=5.5.4",
    "silverstripe/cms": "~3.6.0",
    "silverstripe/framework": "~3.6.0",
    "silverstripe/userforms": "~4.2",
    "silverstripe/blog": "~2.4",
    "silverstripe/widgets": "~1.2",
    "symbiote/silverstripe-gridfieldextensions": "~1.4",
    "silverstripe/dashboard": "~1.1",
    "burnbright/silverstripe-widget-content": "dev-master",
    "unclecheese/betterbuttons": "~1.3",
    "silverstripe/silverstripe-omnipay": "~2.2",
    "omnipay/paypal": "~2.6",
    "silverstripe/dms": "~1.3",
    "drewm/mailchimp-api": "~2.2",
    "symbiote/silverstripe-notifications": "~1.0",
    "deptinternalaffairsnz/silverstripe-date-range-field": "~1.0",
    "sheadawson/silverstripe-zenmessage": "dev-master",
    "silverstripe/queuedjobs": "~2.9"
  },
  "require-dev": {
    "phpunit/PHPUnit": "~3.7",
    "silverstripe/sqlite3": "~1.4",
    "silverstripe-australia/mailcapture": "~2.0",
    "gdmedia/ss-auto-git-ignore": "~1.0",
    "lekoala/silverstripe-debugbar": "~1.0"
  },
  "config": {
    "process-timeout": 600
  },
  "scripts": {
    "post-update-cmd": "GDM\\SSAutoGitIgnore\\UpdateScript::Go"
  },
  "prefer-stable": true,
  "minimum-stability": "dev"
}

```

**Comment by nspyke** [comment]
So deleting my cookies fixes it.
Not sure how a member is logged in, but then not

**Comment by robbieaverill** [comment]
Hi @nspyke that's very strange. Glad it's resolved for you. I've removed the difference of using `Member::currentUserID` and `Member::currentUser` to ensure that we're only using the return value of `Member::currentUser` instead. This won't have any visible difference of behaviour, but should prevent your issue from happening to somebody else.

Fix at #84.

Thanks for reporting!

**Comment by robbieaverill** [comment]
Fixed with #84 and released in 1.1.0
